.github/pull_request_template.md

```diff
@@ -1,10 +1,9 @@
+(Please add to the PR name the issue/s that this PR would close if merged by using a [Github](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) keyword. Example: `closes #999`. If your PR is made by a single commit, please add that clause in the commit too. This is all required to automate the closure of related issues.)
```

Comment:
We could just say "write `closes #123` in your first commit's message". That handles all cases